datadog/resource_datadog_dashboard.go

```diff
@@ -75,17 +75,24 @@ func resourceDatadogDashboard() *schema.Resource {
 				Optional:    true,
 				Description: "The description of the dashboard.",
 			},
+			"url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The URL of the dashboard.",
+			},
 			"is_read_only": {
```

Comment:
Lets mark that this field conflicts with `restricted_roles`.

Reply:
`restricted_roles` is marked as conflicting with `is_read_only`; do you mean "mark both fields as conflicting with each other", or "mark `is_read_only` instead of `restricted_roles`"?

Reply:
Sorry yeah I meant marking both fields as conflicting with one another